src/libraries/Microsoft.PowerFx.Core/Texl/Intellisense/SuggestionHandlers/ErrorNodeSuggestionHandlerBase.cs

```diff
@@ -23,6 +23,12 @@ internal override bool TryAddSuggestionsForNodeKind(IntellisenseData.Intellisens
                 // ThisItemProperties only in the context of thisItem.
                 var curNode = intellisenseData.CurNode;
 
+                // Do not provide error suggestions in string interpolation scenario
```

Comment:
can you add a bit more detail here? Basically, clarify that if the parent is a stringinterp node and not an island, this node is always a partial string literal, and we shouldn't provide suggestions in string literals.